fuzz_test.yml

```diff
@@ -0,0 +1,13 @@
+- name: awkward_BitMaskedArray_to_IndexedOptionArray
+  args:
+    - {name: toindex, type: "List[int64_t]", dir: out}
+    - {name: frombitmask, type: "Const[List[uint8_t]]", dir: in, role: BitMaskedArray-mask}
+    - {name: bitmasklength, type: "int64_t", dir: in, role: default}
+    - {name: validwhen, type: "bool", dir: in, role: BitMaskedArray-valid_when}
+    - {name: lsb_order, type: "bool", dir: in, role: BitMaskedArray-lsb_order}
+  inputs:
+    - toindex: []
```

Comment:
I'd suggest you try to use `roles` here:
https://github.com/scikit-hep/awkward-1.0/blob/f5b298524afddb781fa8ef6792257b494452e318/kernel-specification.yml#L8028-L8032